.github/workflows/check-local-changes.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: ./gradlew generatePackageInfos
 
       - name: Gen patches and ATs
-        run: ./gradlew :neoforge:genPatches :neoforge:generateAccessTransformers
+        run: ./gradlew :neoforge:genPatches :neoforge:generateAccessTransformers :neoforge:generateFinalizeSpawnTargets
```

Comment:
I would now add a task to generate all mc jar-based things.

Reply:
Well there's only two so it sounds a bit premature. :P